app/builders/v2/report_builder.rb

```diff
@@ -96,10 +96,9 @@ def agent_metrics
 
   def conversations
     @open_conversations = scope.conversations.where(account_id: @account.id).open
-    first_response_count = @account.reporting_events.where(name: 'first_response', conversation_id: @open_conversations.pluck('id')).count
     metric = {
       open: @open_conversations.count,
-      unattended: @open_conversations.count - first_response_count
+      unattended: @open_conversations.where(first_reply_created_at: nil).count
```

Comment:
Should we move this to a scope so ensure both the places will stay consistent in calculation ?